packages/wallet/src/rollup-state-solver.ts

```diff
@@ -101,10 +103,12 @@ export class DefaultRollupStateSolver implements RollupStateSolver {
           decider: this.merkleInclusionDecider,
           input: {
             merkleProof: {
-              rootHash: Buffer.from(stateReceipt.stateRoot),
+              rootHash: Buffer.from(stateReceipt.stateRoot, 'hex'),
```

Comment:
I didn't realize `hexStrToBuf(...)` is a thing. We should probably use that everywhere. 

ref: https://github.com/plasma-group/pigi/pull/460/files#diff-3c6a9a75c3d9298f45efe6c6dab74ecdR388

Reply:
Yeah I noticed going through the code that you might not have realized we had that